lib/matplotlib/backends/backend_wx.py

```diff
@@ -1337,6 +1360,26 @@ def _load_bitmap(filename):
     return bmp
 
 
+_FRAME_ICON = None
```

Comment:
I'm not that fond of this Gloabal icon.
It seems to be there just to save loading again the same image. 
Unless we are short on memory or it does impose a penalty on performance, I would say, just load it again

Reply:
It's actually a set of bitmaps with different sizes. As actually there are only two, I'm fine with reloading. I've just pushed the modification.